pkg/dashboard/templates/email.gohtml

```diff
@@ -0,0 +1,27 @@
+{{ define "email" }}
+<aside class="email-box" id="email-box">
+  <details open>
+    <summary class="--hideMarker">
+      <p class="email-box__toggle-title">
+        Add cost estimates to your Goldilocks dashboard!
+        <i aria-hidden="true" class="fas fa-fw fa-angle-down --showWhenClosed"></i
+        ><i aria-hidden="true" class="fas fa-fw fa-angle-up --showWhenOpen"></i>
+      </p>
+    </summary>
+
+    <form class="email-box__content">
+      <p>Enter your email to get a free estimate of the cost impact of applying Goldilocks’ recommendations.</p>
+      <label class="email-box__email-label">
+        <input id="email-box__email-input" class="email-box__email-input" type="email" />
+        <span id="email-box__email-label-content">Email</span>
+      </label>
+      <label class="email-box__checkbox-label">
+        <input type="checkbox" id="email-box__checkbox">
+        <span class="email-box__checkbox-content">By entering your email address, you acknowledge that we may use it as our</span>
+      </label>
+      <span class="email-box__privacy-wrapper"><a class="email-box__privacy" href="https://www.fairwinds.com/privacy-policy">Privacy Policy</a> describes.</span>
```

Comment:
This change helps make the link much easier to interact with, thank you! However, it's also introduced a new accessibility bug.

Visually, the sentence associated with the checkbox is complete, but someone using a screen reader will hear that there's a checkbox, and that the text associated with it says "By entering your email address, you acknowledge that we may use it as our" (with the sentence left incomplete). The full text, without the link, needs to be inside the `<label>` for screen readers to pick it up correctly.

One way to make this work both visually and as announced by screen readers could look like this:
```html
<label class="email-box__checkbox-label">
	<input type="checkbox" id="email-box__checkbox">
	<span class="email-box__checkbox-content">By entering your email address, you acknowledge that we may use it as our Privacy Policy describes.</span>
</label>
<p class="email-box__privacy-wrapper">View our <a class="email-box__privacy" href="https://www.fairwinds.com/privacy-policy">Privacy Policy</a>.</p>
```

Reply:
I updated the code. Thank you for your solution!

Reply:
You're very welcome! Glad I could help!